pdal/pdal_config.cpp

```diff
@@ -93,6 +93,9 @@ bool hasFeature(Feature f)
         enabled = true;
 #endif
         break;
+    case Feature::LAS_START:
```

Comment:
LAS_START? Would we not eventually add start capability to other drivers?

Reply:
If you're building a client, you can use this to check if PDAL has this support without resorting to version numbers.